package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "eslint . && eslint --ext md documentation && prettier --check '**/*.{js,md}'",
     "generate-site": "generate-site --require ./bootstrap-unexpected-markdown.js",
     "update-examples": "generate-site --require ./bootstrap-unexpected-markdown.js --update-examples",
-    "preversion": "test -n \"${IS_MAKE_RELEASE}\" || (echo Please run make release instead && exit 1)"
+    "preversion": "(test -n \"${IS_MAKE_RELEASE}\" || (echo Please run make release instead && exit 1)) && offline-github-changelog --next=${npm_package_version} > CHANGELOG.md && git add CHANGELOG.md"
```

Comment:
@papandreou FYI https://npm.community/t/npm-version-preversion-npm-package-version/1406 && https://github.com/npm/cli/pull/51. This might fail from npm 7

Reply:
Ah, thanks. I guess we should switch to `version`, then?

Reply:
Yes, it seems to fit this use case 🙂

Reply:
Fixed in d21792db4cd1e34cc890bda360d51afd42df5f88, thanks :+1: